cmd/kubelet/app/options/container_runtime.go

```diff
@@ -71,9 +71,6 @@ type ContainerRuntimeOptions struct {
 	// and overrides the default MTU for cases where it cannot be automatically
 	// computed (such as IPSEC).
 	NetworkPluginMTU int32
-	// NetworkPluginDir is the full path of the directory in which to search
-	// for network plugins (and, for backwards-compat, CNI config files)
-	NetworkPluginDir string
```

Comment:
do we need to remove the binDir as well ?

Reply:
`NetworkPluginDir` is both for config and bin. It has been replaced with `CNIConfDir` for config and `CNIBinDir` for bin, there is no additional binDir needed to be remove.